src/awsexplorer/activation.ts

```diff
@@ -124,9 +124,12 @@ async function registerAwsExplorerCommands(
     )
 
     context.subscriptions.push(
-        vscode.commands.registerCommand('aws.refreshAwsExplorer', async () => {
-            recordAwsRefreshExplorer()
+        vscode.commands.registerCommand('aws.refreshAwsExplorer', async (emitTelemetry: boolean = true) => {
```

Comment:
let's name this `passive`, for consistency with `src/credentials/loginManager.ts`, as well as telemetry's own parameter names . That also expresses the semantics a bit better.

```suggestion
        vscode.commands.registerCommand('aws.refreshAwsExplorer', async (passive: boolean = false) => {
```